supplementary_style_guide/glossary_terms_conventions/product_conventions/jboss-eap.adoc

```diff
@@ -1,10 +1,901 @@
+
+*Use it*: yes
+
+*Incorrect forms*: GUI, HTTP interface
+
+*See also*: xref:http-interface[HTTP interface]
+
+[discrete]
+[[messaging-heading]]
+==== Messaging (noun)
+*Description*: Use "Messaging" when referring to the xref:messaging-activemq[messaging-activemq] subsystem in titles and headings. Capitalize the word. See xref:messaging-activemq[messaging-activemq] entry for the correct usage in general text. See xref:messaging-activemq-management[Messaging - ActiveMQ] entry for the correct usage when referring to the messaging-activemq subsystem in the management console.
+
+*Use it*: yes
+
+*Incorrect forms*: messaging, messaging-activemq, Messaging-activemq, Messaging-Activemg, Messaging-ActiveMQ
+
+*See also*: xref:messaging-activemq[messaging-activemq], xref:messaging-activemq-management[Messaging - ActiveMQ]
+
+[discrete]
+[[messaging-activemq]]
+==== messaging-activemq (noun)
+*Description*: The "messaging-activemq" subsystem is used to configure messaging in JBoss EAP. Write in lower case as two words separated by a hyphen. See xref:messaging-heading[Messaging] entry for the correct usage in headings. See xref:messaging-activemq-management[Messaging - ActiveMQ] entry for the correct usage when referring to the messaging-activemq subsystem in the management console.
+
+*Use it*: yes
+
+*Incorrect forms*: messaging, Messaging
+
+*See also*: xref:messaging-heading[Messaging], xref:messaging-activemq-management[Messaging - ActiveMQ], xref:messaging-subsystem[messaging subsystem]
+
+[discrete]
+[[messaging-activemq-management]]
+==== Messaging - ActiveMQ (noun)
+*Description*: Use "Messaging - ActiveMQ" when describing the messaging-activemq subsystem in the management console. Write as two capitalized words separated by two spaces and a hyphen. Ensure that "MQ" is also in uppercase.
+
+*Use it*: yes
+
+*Incorrect forms*: Any form omitting the spaces, hyphen or correct capitalization
+
+*See also*: xref:messaging-activemq[messaging-activemq], xref:messaging-heading[Messaging], xref:messaging-subsystem[messaging subsystem]
+
+[discrete]
+[[messaging-subsystem]]
+==== messaging subsystem (noun)
+*Description*: "messaging subsystem" is an acceptable generic term for referring to the messaging-activemq subsystem. However, see xref:messaging-activemq-management[ActiveMQ - Management] entry for the correct usage when referring to the messaging-activemq subsystem in the management console.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:messaging-activemq[messaging-activemq], xref:messaging-heading[Messaging], xref:messaging-activemq-management[Messaging - ActiveMQ]
+
+[discrete]
+[[microsoft-windows]]
+==== Microsoft Windows (noun)
+*Description*: Do not use "Microsoft Windows" to refer to the Microsoft Windows Server product or Windows-specific commands and scripts such as standalone.bat. when referring to the modcluster subsystem in titles and headings. See xref:windows-server[Windows Server] entry for the correct usage.
```

Comment:
Is there an extra period after `standalone.bat`?

Reply:
Fixed